generate-concourse-environment.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -eu
+
+RED='\033[0;31m'
+CYAN='\033[0;36m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+PURPLE='\033[0;35m'
+NC='\033[0m'
+
+if [[ -z $CG_PIPELINE ]]
+then
+  echo -e "${RED}ERROR${NC} Please set a ${YELLOW}\$CG_PIPELINE${NC} variable pointing to a clone of ${YELLOW}https://github.com/18F/cg-pipeline-tasks${NC}"
+  echo -e "eg, ${PURPLE}CG_PIPELINE=~/dev/cg-pipeline-tasks ./generate-k8s-release-certs.sh"
+  exit 98
+fi
+
+if [[ -z $SECRETS_BUCKET ]]
+then
+  echo -e "${RED}ERROR${NC} Please set a ${YELLOW}\$SECRETS_BUCKET${NC} with the name of the ${YELLOW}s3 bucket where secrets are stored${NC}"
+  echo -e "eg, ${PURPLE}CG_PIPELINE=~/dev/cg-pipeline-tasks ./generate-k8s-release-certs.sh"
+  exit 98
+fi
+
+if [[ -z $CI_ENV ]]
+then
+  echo -e "${RED}ERROR${NC} Please set a ${YELLOW}\$CI_ENV${NC} with the name of the ${YELLOW}concourse target where the secret-rotationpipeline is stored${NC}"
+  echo -e "eg, ${PURPLE}CG_PIPELINE=~/dev/cg-pipeline-tasks ./generate-k8s-release-certs.sh"
+  exit 98
+fi
+
+# create a combined secrets file
+echo "foo: bar" > secrets-combined.yml
```

Comment:
Can we make this initial key a bit more random so we can get rid of any race condition where foo might be a valid key?  Also, this will make sure that if valid later, it doesn't get pruned in the later merge.